tools/mage/go.go

```diff
@@ -71,6 +71,10 @@ func outputGo(cmd string, args ...string) (string, error) {
 	return buf.String(), nil
 }
 
+func runGoMute(args ...string) error {
```

Comment:
This is only ever used once in the whole codebase, remove and just use `execGo(nil, os.Stderr, "run", args...)` explicitly